masterlist.yaml

```
@@ -11505,6 +11505,25 @@ plugins:
       - crc: 0x606B1EC2
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'Reaction( - Additions Only)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/46676/'
+        name: 'Reaction'
+    req: [ *SilentVoice ]
```

Looks like someone made [voice files](https://www.nexusmods.com/oblivion/mods/52417) for the mod, so the requirement could be hidden if they're installed:
```suggestion
    req:
      - <<: *SilentVoice
        condition: 'not file("OBSE/Plugins/voicefile_redirector.dll") and not file("sound/voice/Reaction( - Additions Only)?\.esp")'
```